lib/allure_report_publisher/lib/providers/gitlab.rb

```diff
@@ -62,24 +62,55 @@ def update_pr_description
       #
       # @return [void]
       def add_comment
-        unless comment
+        client.delete_merge_request_discussion_note(project, mr_iid, discussion.id, alert_comment.id) if alert_comment
+
+        if discussion
```

Comment:
I think we need to make this flow togglable and off by default. This would technically be a breaking change, if someone has a setup with automatic merging with tests results being ignored (very bad example, but hey, we ourselves ignore test results 😆 ), this would break that flow because now a not resolved discussion would start appearing.

This cli already has too many cli args probably, but I think still needs to be behind one. Something like `--unresolved-discussion-on-failure`.